include/tvm/relay/base.h

```diff
@@ -93,7 +93,7 @@ class IdNode : public Object {
    *  this only acts as a hint to the user,
    *  and is not used for equality.
    */
-  std::string name_hint;
+  String name_hint;
```

Comment:
per https://github.com/apache/incubator-tvm/issues/5490#issuecomment-624408382 every update to the IRNode object needs to have a backward compatible update code